src/cloud/components/Props/Pickers/AssigneeSelect.tsx

```diff
@@ -16,13 +16,15 @@ interface AssigneeSelectProps {
   update: (value: string[]) => void
   isLoading: boolean
   readOnly: boolean
+  isErrored?: boolean
```

Comment:
we are essentially displaying the picker style rather than the cell style in the actual cell. 
Also, I feel like error states will most likely end up having specific behaviour on pcikers based on what we want ( like for example having only a delete option etc.. )